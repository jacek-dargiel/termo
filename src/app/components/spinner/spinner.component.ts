import { Component } from '@angular/core';

@Component({
    selector: 'termo-spinner',
    templateUrl: './spinner.component.html',
    styleUrls: ['./spinner.component.scss']
})
export class SpinnerComponent {

  constructor() { }

}
