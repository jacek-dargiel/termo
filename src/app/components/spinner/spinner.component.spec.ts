import { ComponentFixture, TestBed } from '@angular/core/testing';

import { SpinnerComponent } from './spinner.component';
import { provideZonelessChangeDetection } from '@angular/core';

describe('SpinnerComponent', () => {
  let component: SpinnerComponent;
  let fixture: ComponentFixture<SpinnerComponent>;

  beforeEach(() => {
    TestBed.configureTestingModule({
      imports: [ SpinnerComponent ],
      providers: [provideZonelessChangeDetection()]
    })
    .compileComponents();
  });

  beforeEach(() => {
    fixture = TestBed.createComponent(SpinnerComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
  });

  it('should create', () => {
    expect(component).toBeTruthy();
  });

  it('should render spinner', () => {
    expect(fixture).toMatchSnapshot();
  });
});
