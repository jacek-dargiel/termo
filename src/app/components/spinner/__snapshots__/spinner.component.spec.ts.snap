// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`SpinnerComponent should render spinner 1`] = `
<termo-spinner>
  <div
    class="spinner"
  >
    <div
      class="rect1"
    />
    <div
      class="rect2"
    />
    <div
      class="rect3"
    />
    <div
      class="rect4"
    />
    <div
      class="rect5"
    />
  </div>
</termo-spinner>
`;
